src/app/forms/form-layout/page.tsx

```diff
@@ -1,17 +1,40 @@
+'use client';
+import { useSelector, useDispatch } from 'react-redux';
+import { updateContactForm, updateSignInForm, updateSignUpForm } from '@/redux/forms/formSlice';
 import Breadcrumb from "@/components/Breadcrumbs/Breadcrumb";
-
 import { Metadata } from "next";
 import DefaultLayout from "@/components/Layouts/DefaultLayout";
 import SelectGroupOne from "@/components/SelectGroup/SelectGroupOne";
 import Link from "next/link";
+import { RootState } from '@/redux/store';
 
-export const metadata: Metadata = {
-  title: "Next.js Form Layout | TailAdmin - Next.js Dashboard Template",
-  description:
-    "This is Next.js Form Layout page for TailAdmin - Next.js Tailwind CSS Admin Dashboard Template",
-};
+// export const metadata: Metadata = {
```

Comment:
Add metadata to all pages as mentioned here: `https://stackoverflow.com/questions/76445050/not-able-to-change-the-title-with-metadata-when-using-use-client-in-next-js`